lib/linter-elm-make.js

```diff
@@ -7,7 +7,7 @@ module.exports = {
     elmMakeExecutablePath: {
       title: 'The elm-make executable path.',
       type: 'string',
-      default: 'elm-make',
+      default: '/usr/local/bin/elm-make',
```

Comment:
Also, I think this might break initial functionality for at least some people. I just checked my setup, and my `elm-make` executable is in `$HOME/.nvm/versions/node/v4.2.1/bin/elm-make`, which is part of my PATH when I start Atom. The current default value of `elm-make` would work with that (since it's in the path), where changing the default path would mean that I had to explicitly enter the location before the plugin would work.